code/contracts/ONEWallet.sol

```diff
@@ -206,12 +214,12 @@ contract ONEWallet {
         (uint32 ct, bool completed) = _findCommit(hash);
         require(ct > 0, "Cannot find commit for this transaction");
         uint32 counter = ct / interval - t0;
-        require(counter == index, "Provided index does not match commit timestamp");
+        require(counter == index, "Provided index does not match committed timestamp");
         uint8 expectedNonce = nonces[counter];
-        require(nonce >= expectedNonce, "Nonce is too low");
+        require(nonce >= expectedNonce, "Nonce is too low"); // IH: should not be here equality? One commit object is associated with the one nonce (it is part of commited hash)
```

Comment:
Should be `>=` because nonce is incremented during reveal after passing `_revealPreCheck`. Nonce helps tracking the number of proofs already used within one interval. It starts with 0.

Reply:
OK